packages/@react-native-windows/cli/package.json

```diff
@@ -74,5 +74,6 @@
       "patch"
     ]
   },
-  "promoteRelease": true
+  "promoteRelease": true,
+  "windowsOnly": true
```

Comment:
nit: Maybe someone incorporate that this is only for test (as we still build on mac) i.e. "testOnWindowsOnly"

Reply:
I went back and forward on that. Whether to test on Windows maps to "do we use this on anything other than Windows", which made me think to make it more generic. That also makes it less clear if you see it in a package.json though, as to what it affects in practice.

Reply:
Going to merge this as is, because I keep on going back and forward, and this probably isn't worth splitting hairs over.